zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1497,7 +1501,8 @@ and [SASL authentication for ZooKeeper](https://cwiki.apache.org/confluence/disp
 * *sslQuorum* :
     (Java system property: **zookeeper.sslQuorum**)
     **New in 3.5.5:**
-    Enables encrypted quorum communication. Default is `false`.
+    Enables encrypted quorum communication. Default is `false`. When enabling this feature, please also consider enabling *leader.closeSocketAsync*
+    and *learner.closeSocketAsync* to avoid issues associated with the potentially long socket closing time when shutting down an SSL connection.
```

Comment:
I haven't found `learner.closeSocketAsync` in the code. Did I miss something, or should ve not mention that property here?

Reply:
it's in PR https://github.com/apache/zookeeper/pull/1115, which hasn't been merged yet.